libexec/python/utils.py

```diff
@@ -172,19 +172,22 @@ def api_get(url,data=None,default_header=True,headers=None,stream=None,return_re
 
     return stream
 
+
 def basic_auth_header(username, password):
     '''basic_auth_header will return a base64 encoded header object to
     generate a token
     :param username: the username
     :param password: the password
     '''
     s = "%s:%s" % (username, password)
+    logger.debug("AUTH: Input username/password: %s", s)
```

Comment:
Oh, I thought you meant to add them while investigating our authentication bug only. I didn't think you wanted to add the debug prints for everyone.